website/docs/docs/cloud/cloud-cli-installation.md

```diff
@@ -145,43 +144,47 @@ Advanced users can configure multiple projects to use the same Cloud CLI executa
 
 <TabItem value="pip" label="Existing dbt Core users (pip)">
 
+If you already have dbt Core installed, you can install the dbt Cloud CLI using pip. Here are some considerations:
```

Comment:
We say below to not use pip, so I would say "If you already have dbt Core installed, the dbt Cloud CLI may conflict. Here are some considerations:"